env.example

```diff
@@ -3,3 +3,6 @@ NODE_ENV=development
 
 # PORT is the port used by the web server
 PORT=8080
+
+# REDIS_URL specifies Redis server info
+REDIS_URL='redis://127.0.0.1:6379'
```

Comment:
Let's remove the quotes so that it'll be REDIS_URL=redis://127.0.0.1:6379 and add a new line after

Reply:
Removed the quotes an added new line.